lumicks/pylake/fitting/models.py

```diff
@@ -169,6 +170,43 @@ def marko_siggia_simplified(name):
     )
 
 
+def inverted_marko_siggia_simplified(name):
+    """Markov Siggia's Worm-like Chain model based on only entropic contributions (valid for F << 10 pN). This model
+    has distance as a dependent variable.
+
+    References:
+        1. J. Marko, E. D. Siggia. Stretching dna., Macromolecules 28.26,
+        8759-8770 (1995).
+
+    Parameters
+    ----------
+    name : str
+        Name for the model. This name will be prefixed to the model parameter names.
+    """
+    from .model import Model
+    from .detail.model_implementation import (
+        inverted_simplified_marko_sigga,
+        inverted_simplified_marko_sigga_jac,
+        inverted_simplified_marko_sigga_derivative,
+        inverted_simplified_marko_sigga_equation,
+        inverted_simplified_marko_sigga_equation_tex,
```

Comment:
I feel a little bit confused about the naming of the detail functions for the Marko Siggia models: We have `marko_sigga_ewlc_*`,  `Marko_Siggia_*` and now `inverted_simplified_marko_siggia_*`.

I suggest following the public API names, which means:
- Adding an extra commit to rename the forward simplified functions from `Marko_Siggia_*` to  `marko_siggia_simplified_*`.

- Add a commit to fix the typo in `marko_sigga_ewlc_*` functions

- Change the new ones from `inverted_simplified_marko_siggia_*` to `inverted_marko_siggia_simplified_*`.

Reply:
Yes, this is a legacy remnant. Marko_Siggia was the first model.

I've applied your suggestions :)